source/cluster-wide.rst

```diff
@@ -0,0 +1,99 @@
+Install Percona XtraDB Cluster cluster-wide
+============================================
+
+By default, Percona XtraDB Cluster Operator functions in a specific Kubernetes
```

Comment:
We need to add recommendation that we suggest using one  operator per  namespace but if you need you can use cluster-wide....